tests/accessor_legacy/accessor_api_common_all.h

```diff
@@ -50,8 +52,12 @@ void check_accessor_members(sycl_cts::util::logger &log,
 
   using acc_t = sycl::accessor<T, dims, mode, target, placeholder>;
 
+  using t_type = typename std::conditional<mode == sycl::access_mode::read,
+                                           const T, T>::type;
```

Comment:
```suggestion
  using acc_t = sycl::accessor<T, dims, mode, target, placeholder>;

  using t_type = typename std::conditional_t<mode == sycl::access_mode::read,
                                           const T, T>;
```

Reply:
You can get rid of `typename` in that case as well.

Reply:
Done.